docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -0,0 +1,281 @@
+        </listitem>
+
+        <listitem>
+          <para>Prepare to copy portions of default YAML file
+          (..\helm\hpcc\values.yaml) </para>
+
+          <para>You can use the HPCC-Platform repository or the helm-chart
+          repository. </para>
+
+          <para>https://github.com/hpcc-systems/HPCC-Platform </para>
+
+          <para>https://github.com/hpcc-systems/helm-chart </para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire <emphasis role="bold">roxie:</emphasis>
+          section and paste it into the new TwoRoxies.yaml file.</para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire contents, except the first line, and paste it
+          at the end of the file.</para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">name:</emphasis> and change it to <emphasis
+          role="bold">roxie2</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">name:</emphasis> under <emphasis
+          role="bold">services:</emphasis> and change it to <emphasis
+          role="bold">roxie2</emphasis>.</para>
+        </listitem>
+
+        <listitem>
+          <para>Save the file and close the text editor.</para>
+
+          <para>The resulting TwoRoxies.yaml file should look like this</para>
+
+          <para><emphasis role="bold">Note:</emphasis> The comments have been
+          removed to simplify the example:</para>
+
+          <para><programlisting>roxie:
+- name: roxie
+  disabled: false
+  prefix: roxie
+  services:
+  - name: roxie
+    servicePort: 9876
+    listenQueue: 200
+    numThreads: 30
+    visibility: local
+  replicas: 2  
+  numChannels: 2
+  serverReplicas: 0
+  localAgent: false
+  traceLevel: 1
+  topoServer:
+    replicas: 1
+
+- name: roxie2
+  disabled: false
+  prefix: roxie2
+  services:
+  - name: roxie2
+    servicePort: 9876
+    listenQueue: 200
+    numThreads: 30
+    visibility: local
+  replicas: 2  
+  numChannels: 2
+  serverReplicas: 0
+  localAgent: false
+  traceLevel: 1
+  topoServer:
+    replicas: 1
+</programlisting></para>
+        </listitem>
+      </orderedlist>
+
+      <para><emphasis role="bold">Deploy using the new custom configuration
+      chart.</emphasis></para>
+
+      <orderedlist>
+        <listitem>
+          <para>Open a terminal and navigate to the <emphasis
+          role="bold">helm</emphasis> folder of the repository you
+          downloaded.</para>
+        </listitem>
+
+        <listitem>
+          <para>Deploy your HPCC Systems Platform, adding the new
+          configuration to your command:</para>
+
+          <para><programlisting>helm install mycluster hpcc/ -f TwoRoxies.yaml</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Open ECL Watch. </para>
```

Comment:
it is probably worth noting that they may have to wait a few minutes at this point.
And/or if they do not see the 2 targets, that they need check the status of deployments and pods in Kubernetes (e.g. because might be pending on downloading an image or similar). To much to cover in this document, but a small note and/or reference to something would be good.